pkgs/development/python-modules/fastecdsa/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, gmp
+}:
+
+buildPythonPackage rec {
+  pname = "fastecdsa";
+  version = "2.1.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "d0772f7fe243e8a82d33e95c542ea6cc0ef7f3cfcced7440d6defa71a35addfa";
+  };
+
+  buildInputs = [ gmp ];
+
+  # skip tests, most of them require being online to download test vectors
+  doCheck = false;
```

Comment:
There are two test files specifically that require downloading third-party files: https://github.com/AntonKueltz/fastecdsa/search?q=urlopen.

I believe it should be feasible to get the tests up and running using `pytestCheckHook` and `pytestFlags = [ "--ignore=fastecdsa/tests/test_wycheproof_vectors.py" "--ignore=fastecdsa/tests/test_rfc6979_ecdsa.py" ]`

Reply:
I did this, but now other tests fail with:

```
ImportError while importing test module '/build/fastecdsa-2.1.5/fastecdsa/tests/encoding/test_asn1.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
fastecdsa/tests/encoding/test_asn1.py:5: in <module>
    from fastecdsa.keys import export_key, import_key, gen_keypair
fastecdsa/keys.py:7: in <module>
    from .ecdsa import verify
fastecdsa/ecdsa.py:5: in <module>
    from fastecdsa import _ecdsa
E   ImportError: cannot import name '_ecdsa' from 'fastecdsa' (/build/fastecdsa-2.1.5/fastecdsa/__init__.py)
```

Any ideas why? `/build/fastecdsa-2.1.5` looks spurious, but not sure where this is from ...

Reply:
I guess that's because you are not building this: https://github.com/AntonKueltz/fastecdsa/tree/master/src

Reply:
That is mentioned in the setup.py and the following works if I skip the tests: `from fastecdsa import _ecdsa`

Reply:
Right, I'm not sure then.

Reply:
I rebased this on top of the current master and now it works - included in c28e12b6b552b2b3fe48f2970eb03c3d2a277828

Reply:
What is included? If the tests still don't work but the resulting package works, add a comment why you disabled them.

Reply:
Ah, sorry. Brainfart - tests disabled and comment added in f70ecb65e71837e1029796d83af253a904ab38b6